model/sshable.rb

```diff
@@ -88,7 +88,7 @@ def connect
     end
 
     # Cache miss.
-    sess = Net::SSH.start(host, "rhizome", **COMMON_SSH_ARGS.merge(key_data: keys.map(&:private_key)))
+    sess = Net::SSH.start(host, unix_user, **COMMON_SSH_ARGS.merge(key_data: keys.map(&:private_key)))
     Thread.current[:clover_ssh_cache][host] = sess
```

Comment:
Although safe in the case of where Sshable never has its unix_user mutated and `host` has a unique constraint, something about this gives me the heebie jeebies, that a session ends up in the cache under a different user with different permissions, and gets pulled out again without a proximate check that the user the connection is established with indeed matches.

Heebie jeebies is not quite a good enough reason to make things more a lot more complex, but if it falls out of the design neatly, you could change the cache definition to `Thread.current[:clover_ssh_cache][[host, user]]` (or even a struct with the two values) to form a hash key that embeds the account the session was logged in on. This can be an enhancement on my proposal to split this patch and submit it for rapid merging.